packages/core/src/layout/Header/Header.tsx

```diff
@@ -200,8 +200,10 @@ export const Header: FC<Props> = ({
   type,
   typeLink,
 }) => {
-  const theme = useContext(PageThemeContext);
-  const classes = useStyles({ backgroundImage: theme.backgroundImage });
+  const backstageTheme = useTheme<BackstageTheme>();
+  const classes = useStyles({
+    backgroundImage: backstageTheme.page.backgroundImage,
```

Comment:
Now that this is a theme you can access it within `makeStyles`, so no need for the extra access here

Reply:
Good catch 😼 Would love to make it simple